Casks/k/keybase.rb

```diff
@@ -17,8 +17,9 @@
 
   livecheck do
     url "https://prerelease.keybase.io/update-darwin-#{arch}prod-v2.json"
-    strategy :page_match do |page|
-      match = page.match(/Keybase[._-]v?(\d+(?:\.\d+)+)[._-](\d+)%2B([0-9a-f]+)\.zip/i)
+    regex(/(\d+(?:\.\d+)+)[._-](\d+)\+([0-9a-f]+)/i)
+    strategy :json do |json, regex|
+      match = json["version"]&.match(regex)
       next if match.blank?
 
       "#{match[1]},#{match[2]},#{match[3]}"
```

Comment:
```suggestion
    strategy :json do |json|
      json["version"]&.tr("-+", ",")
```

We could substitute the `-` and `+` in the version string with `,`. It achieves the same goal but it's simpler and would continue to work if the version format changes to omit one or more of the suffixes (we could do the same thing with optional capture groups in the regex/`strategy` block but it's more involved).